### ISSUE TITLE
Support VS Code 1.53.0

### ISSUE THREAD

**Issue by apupier** [redhat-developer/vscode-extension-tester#221]
* need to add the 1.52.0 in test matrix

* I think that errors comes from VS Code extension tester: https://app.circleci.com/pipelines/github/camel-tooling/camel-lsp-client-vscode/263/workflows/c7ace814-4c70-4a45-b6b7-2d2917c9fc57/jobs/296 https://issues.redhat.com/browse/FUSETOOLS2-987 it seems to be a problem with non native dialog to open a file

```
 NoSuchElementError: no such element: Unable to locate element: {"method":"css selector","selector":".editor-instance"}
 ```
 
* https://github.com/redhat-developer/vscode-extension-tester/pull/222/checks?check_run_id=1862073028#step:5:413
```
1) BottomBarPanel
       can be maximized and restored:

      AssertionError: expected 256 to equal 716
      + expected - actual

      -256
      +716
      
      at /home/runner/work/vscode-extension-tester/vscode-extension-tester/test/test-project/out/src/test/bottomBar/bottomBarPanel-test.js:40:35
      at Generator.next (<anonymous>)
      at fulfilled (out/src/test/bottomBar/bottomBarPanel-test.js:5:58)
      at runMicrotasks (<anonymous>)
      at processTicksAndRejections (internal/process/task_queues.js:97:5)
```
* https://github.com/redhat-developer/vscode-extension-tester/pull/222/checks?check_run_id=1862073028#step:5:428
```
2) ProblemsView
       "before all" hook for "get all markers works":
     TimeoutError: Waiting until element is visible
Wait timed out after 5022ms
      at /home/runner/work/vscode-extension-tester/vscode-extension-tester/node_modules/selenium-webdriver/lib/promise.js:2201:17
      at ManagedPromise.invokeCallback_ (/home/runner/work/vscode-extension-tester/vscode-extension-tester/node_modules/selenium-webdriver/lib/promise.js:1376:14)
      at TaskQueue.execute_ (/home/runner/work/vscode-extension-tester/vscode-extension-tester/node_modules/selenium-webdriver/lib/promise.js:3084:14)
      at TaskQueue.executeNext_ (/home/runner/work/vscode-extension-tester/vscode-extension-tester/node_modules/selenium-webdriver/lib/promise.js:3067:27)
      at /home/runner/work/vscode-extension-tester/vscode-extension-tester/node_modules/selenium-webdriver/lib/promise.js:2927:27
      at /home/runner/work/vscode-extension-tester/vscode-extension-tester/node_modules/selenium-webdriver/lib/promise.js:668:7
      at runMicrotasks (<anonymous>)
      at processTicksAndRejections (internal/process/task_queues.js:97:5)
  From: Task: Waiting until element is visible
      at scheduleWait (/home/runner/work/vscode-extension-tester/vscode-extension-tester/node_modules/selenium-webdriver/lib/promise.js:2188:20)
      at ControlFlow.wait (/home/runner/work/vscode-extension-tester/vscode-extension-tester/node_modules/selenium-webdriver/lib/promise.js:2517:12)
      at Driver.wait (/home/runner/work/vscode-extension-tester/vscode-extension-tester/node_modules/selenium-webdriver/lib/webdriver.js:934:29)
      at EditorGroup.<anonymous> (/home/runner/work/vscode-extension-tester/vscode-extension-tester/page-objects/out/components/AbstractElement.js:50:36)
      at Generator.next (<anonymous>)
      at /home/runner/work/vscode-extension-tester/vscode-extension-tester/page-objects/out/components/AbstractElement.js:8:71
      at new Promise (<anonymous>)
      at __awaiter (/home/runner/work/vscode-extension-tester/vscode-extension-tester/page-objects/out/components/AbstractElement.js:4:12)
      at EditorGroup.wait (/home/runner/work/vscode-extension-tester/vscode-extension-tester/page-objects/out/components/AbstractElement.js:49:16)
      at EditorView.<anonymous> (/home/runner/work/vscode-extension-tester/vscode-extension-tester/page-objects/out/components/editor/EditorView.js:138:157)
```
* https://github.com/redhat-developer/vscode-extension-tester/pull/222/checks?check_run_id=1862073028#step:5:452
```
3) ProblemsView
       "after all" hook for "filtering works":
     TypeError: Cannot read property 'clearText' of undefined
      at /home/runner/work/vscode-extension-tester/vscode-extension-tester/test/test-project/out/src/test/bottomBar/problemsView-test.js:56:22
      at Generator.next (<anonymous>)
      at fulfilled (out/src/test/bottomBar/problemsView-test.js:24:58)
      at runMicrotasks (<anonymous>)
      at processTicksAndRejections (internal/process/task_queues.js:97:5)
```

**Comment by jrichter1** [comment]
I've published a (hopefully) fixed version of the page object modules. Make sure your project is pulling the right versions
 - monaco-page-objects 1.4.1
 - vscode-extension-tester-locators 1.53.0

**Comment by apupier** [comment]
What about releasing a new version of the project that requires these dependencies?

**Comment by apupier** [comment]
it is not fixing the problem with open file fialog https://app.circleci.com/pipelines/github/camel-tooling/camel-lsp-client-vscode/267/workflows/d562842a-205e-44cf-93a5-b0e266639ffa/jobs/299 https://github.com/camel-tooling/camel-lsp-client-vscode/pull/541

**Comment by jrichter1** [comment]
I've published page objects 1.4.2, should have a fix for the open file dialog. Could you try it out? If it works, I can release the whole package.

**Comment by apupier** [comment]
it is going further btu there are other errors https://app.circleci.com/pipelines/github/camel-tooling/camel-lsp-client-vscode/268/workflows/64ac55f7-d2c2-4430-abf8-f7023cb9df35/jobs/300

**Comment by jrichter1** [comment]
Right, I've ended up trying out your code. The problem is that the new title bar menu handling is a bit slower because it waits for the whole context menu to load when a menu item is clicked. That can add about a second of test time per invocation. On the other hand, the `menuItemReady` should no longer be of any value.

All in all, your problems are now caused by `_clean` running longer than 2 seconds.
